### PR TITLE
Configure application as web app, add UserDto and UserController

### PR DESCRIPTION
**Pull request by OleksiiTromsa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +54,33 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.17</version>
         </dependency>
+        <dependency>
```
Comment (jah701) [inline pom.xml:57]: I suppose we aint need this dependencies in this h\w?

**Review by jah701 — APPROVED** [review]
Well done!

File: pom.xml (hunk 1 above)
Comment (OleksiiTromsa) [reply pom.xml:57]: Yes, we don't need it, but I've tested examples with jsp and servlets and included these dependencies

Comment (Sofasmile) [reply pom.xml:57]: we can remove them

File: src/main/java/spring/intro/config/WebConfig.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"spring.intro.controllers"})
+public class WebConfig {
+    @Bean
+    public InternalResourceViewResolver resolver() {
```
Comment (Sofasmile) [inline src/main/java/spring/intro/config/WebConfig.java]: we can remove this bean

File: src/main/java/spring/intro/dao/impl/UserDaoImpl.java [hunk 3]
```diff
@@ -44,4 +44,15 @@ public User add(User user) {
             return session.createQuery("FROM User ", User.class).getResultList();
         }
     }
+
+    @Override
+    public User getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.createQuery("FROM User WHERE id = :userId", User.class)
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/dao/impl/UserDaoImpl.java]: let's use session get here instead

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**